lib/widgets/content.dart

```diff
@@ -32,33 +32,72 @@ import 'text.dart';
 /// background. For what this is in the message list, see
 /// widgets/message_list.dart.
 class ContentTheme extends ThemeExtension<ContentTheme> {
-  ContentTheme(BuildContext context) :
-    textStylePlainParagraph = TextStyle(
-      inherit: false,
+  factory ContentTheme.light(BuildContext context) {
+    return ContentTheme._(
+      colorCodeBlockBackground: const HSLColor.fromAHSL(0.04, 0, 0, 0).toColor(),
+      colorDirectMentionBackground: const HSLColor.fromAHSL(0.2, 240, 0.7, 0.7).toColor(),
+      colorGlobalTimeBackground: const HSLColor.fromAHSL(1, 0, 0, 0.93).toColor(),
+      colorGlobalTimeBorder: const HSLColor.fromAHSL(1, 0, 0, 0.8).toColor(),
+      colorMathBlockBorder: const HSLColor.fromAHSL(0.15, 240, 0.8, 0.5).toColor(),
+      colorMessageMediaContainerBackground: const Color.fromRGBO(0, 0, 0, 0.03),
+      colorThematicBreak: const HSLColor.fromAHSL(1, 0, 0, .87).toColor(),
+      textStylePlainParagraph: _plainParagraphCommon(context)
+        .apply(color: const HSLColor.fromAHSL(1, 0, 0, 0.15).toColor())
+        .copyWith(debugLabel: 'ContentTheme.textStylePlainParagraph'),
```

Comment:
Is there something subtle `apply` is doing here that's different from another argument to `copyWith`?

Reply:
Oh, no, I don't think so. I think I was just thinking of the `.copyWith(debugLabel` as its own separate step at the end, because we don't want anything to clobber the `debugLabel` we're giving it. But this change wouldn't cause it to be clobbered, so sounds good.